.github/workflows/test.yaml

```
@@ -0,0 +1,13 @@
+---
+name: 'Test'
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: perldocker/perl-tester
+    steps:
+      - uses: actions/checkout@v4
+      - run: make test-t
```

If possible I suggest to run *all* tests

```suggestion
      - run: make test
```

if it's not possible then at least be explicit and don't call this workflow "Test" but something like "Test dynamic tests"

---

I renamed the title.
The other tests are already covered in the other workflow files.